src/pattern-library/components/patterns/prototype/SharedAnnotationsPage.tsx

```diff
@@ -159,6 +159,7 @@ export default function SharedAnnotationsPrototypePage() {
             </p>
           </>
         }
+        level={3}
```

Comment:
In this case, the automatic level would be `2` but I wanted a 3rd-level section.